samples/IdentitySample.Mvc/Migrations/20151110211659_Identity2-3.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Data.Entity.Migrations;
+using Microsoft.Data.Entity.Metadata;
+
+namespace IdentitySample.Mvc.Migrations
+{
+    public partial class Identity23 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
```

Comment:
I may be thinking at it the wrong way, but I am surprised to see all the tables created. It looks more like an initial migration. If this is supposed to be the upgrade migration these should be alters that add a few columns.